packages/vaadin-text-field/src/vaadin-email-field.d.ts

```diff
@@ -1,32 +1,6 @@
-import { TextFieldElement } from './vaadin-text-field.js';
-
 /**
- * `<vaadin-email-field>` is a Web Component for email field control in forms.
- *
- * ```html
- * <vaadin-email-field label="Email">
- * </vaadin-email-field>
- * ```
- *
- * ### Styling
- *
- * See [`<vaadin-text-field>`](#/elements/vaadin-text-field) for the styling documentation.
- *
- * See [Styling Components](https://vaadin.com/docs/latest/ds/customization/styling-components) documentation.
- *
- * @fires {Event} input - Fired when the value is changed by the user: on every typing keystroke, and the value is cleared using the clear button.
- * @fires {Event} change - Fired when the user commits a value change.
- * @fires {CustomEvent} invalid-changed - Fired when the `invalid` property changes.
- * @fires {CustomEvent} value-changed - Fired when the `value` property changes.
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-declare class EmailFieldElement extends TextFieldElement {
-  _createConstraintsObserver(): void;
-}
-
-declare global {
-  interface HTMLElementTagNameMap {
-    'vaadin-email-field': EmailFieldElement;
-  }
-}
-
-export { EmailFieldElement };
+export * from '@vaadin/email-field/src/vaadin-email-field.js';
```

Comment:
Shouldn't we export `.ts` file? Just came to my thought and don't know the correct thing though ...

Reply:
TypeScript correctly resolves `.js` extension and we do not publish `.ts` files, so I think this can be left as is.